utils/fnIt.ts

```diff
@@ -0,0 +1,13 @@
+import { spy } from "ts-mockito";
+
+const getName = <T>(fn: (arg: T) => (...args: any[]) => any ) => {
+    // create an empty instance
+    const dummyInstance = {} as T;
+    // spy it to mark up some properties
+    const dummySpy: T = spy(dummyInstance);
+    // used this 'typed' spy to extract the name
+    return fn(dummySpy).name;
+}
+export function fnIt<T>(fn: (t:T) => (...args:any[]) => any, message : string, test: () => void){
```

Comment:
Since this is the only function in this file (and it has the same name as the file), it's probably appropriate to do a `default` export. This allows for slightly cleaner imports in other files. Not essential though, just pointing it out so you know `default` is there.